composer.json

```diff
@@ -20,11 +20,11 @@
     "guzzlehttp/guzzle": "~6.0"
   },
   "require-dev": {
-    "phpunit/phpunit": "~4.0"
+    "phpunit/phpunit": "5.0"
```

Comment:
Using a fixed constraint like `5.0` will prevent any updates of minor or patch releases. A better way is to use either the [next significant release operator `^` or `~`](https://opencollective.com/tenancy/members/users.json), a [wildcard `5.*`](https://getcomposer.org/doc/articles/versions.md#wildcard-version-range-) or a [range `>= 5.0 < 6`](https://getcomposer.org/doc/articles/versions.md#version-range).

👍

Reply:
didn't work, but we can reconfigure travis to use the vendor's configured phpunit version

with: ./vendor/bin/phpunit (in travis config file)

Reply:
Using the composer installed version makes a lot of sense. But whether it is okay to modify in the travis configuration is a decision a maintainer has to make.